flocker/node/script.py

```diff
@@ -174,6 +174,7 @@ def flocker_dataset_agent_main():
     # Later, construction of this object can be moved into
     # DatasetAgentServiceFactory.get_service where various options passed on
     # the command line could alter what is created and how it is initialized.
+    # XXX: Implement the factory proposed above (here or in a blocking / follow up issue)
```

Comment:
We should talk about whether passing command-line arguments (or specifying a config file) on each node is the way to go for node configuration.  We'd previously discussed centralizing the configuration at the control service but maybe we should delay that work until later since it's not strictly necessary and we're likely to run out of time for nice-to-haves.